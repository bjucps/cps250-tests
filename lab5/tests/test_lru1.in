ape
bat
cat
ape
bat
cat
dog
bat
emu
dog
cat
