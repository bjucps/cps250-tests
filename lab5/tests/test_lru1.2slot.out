ape: MISS (1/2 full; no eviction)
bat: MISS (2/2 full; no eviction)
cat: MISS (2/2 full; evicted 'ape')
ape: MISS (2/2 full; evicted 'bat')
bat: MISS (2/2 full; evicted 'cat')
cat: MISS (2/2 full; evicted 'ape')
dog: MISS (2/2 full; evicted 'bat')
bat: MISS (2/2 full; evicted 'cat')
emu: MISS (2/2 full; evicted 'dog')
dog: MISS (2/2 full; evicted 'bat')
cat: MISS (2/2 full; evicted 'emu')
--------------------------------
FINAL CACHE CONFIGURATION:
--------------------------------
cat
dog
